Should we enable `ui` feature in `server/Cargo.toml` ?

Running this command:

```
➜  trustify git:(main) ✗ cargo check --manifest-path server/Cargo.toml --no-default-features
```

We can see this error:

https://gist.github.com/helio-frota/4d0ca06f35bbdf32a75dd8e53c60f2fb

When enabling and running the command again

`trustify-module-ui = { workspace = true, features = ["ui"] }`

The error was reduced to

```
error[E0609]: no field `ui` on type `InitData`
   --> server/src/lib.rs:249:50
    |
249 |         let ui = Arc::new(UiResources::new(&self.ui)?);
    |                                                  ^^ unknown field
    |
    = note: available fields are: `authenticator`, `authorizer`, `db`, `storage`, `http` ... and 3 others

For more information about this error, try `rustc --explain E0609`.
```

--- a/modules/ui/Cargo.toml
+++ b/modules/ui/Cargo.toml
@@ -7,8 +7,5 @@ edition = "2021"
 actix-web = { workspace = true }
 actix-web-static-files = { workspace = true }
 anyhow = { workspace = true }
-trustify-ui = { workspace = true, optional = true }
+trustify-ui = { workspace = true }
 
-[features]
-default = [ "ui" ]
-ui = ["trustify-ui"]

--- a/modules/ui/src/lib.rs
+++ b/modules/ui/src/lib.rs
@@ -1,5 +1,2 @@
-#[cfg(feature = "ui")]
 pub use trustify_ui::UI;
-
-#[cfg(feature = "ui")]
 pub mod endpoints;

--- a/server/Cargo.toml
+++ b/server/Cargo.toml
@@ -40,6 +40,3 @@ test-context = { workspace = true }
 test-log = { workspace = true, features = ["log", "trace"] }
 trustify-test-context = { workspace = true }
 
-[features]
-default = ["ui"]
-ui = ["trustify-module-ui/ui"]

--- a/server/src/lib.rs
+++ b/server/src/lib.rs
@@ -51,7 +51,7 @@ use trustify_module_graphql::RootQuery;
 use trustify_module_importer::server::importer;
 use trustify_module_ingestor::graph::Graph;
 use trustify_module_storage::{service::dispatch::DispatchBackend, service::fs::FileSystemBackend};
-use trustify_module_ui::{UI, endpoints::UiResources};
+use trustify_module_ui::{endpoints::UiResources, UI};
 
 use utoipa::OpenApi;
 use utoipa_swagger_ui::SwaggerUi;

--- a/server/src/lib.rs
+++ b/server/src/lib.rs
@@ -8,9 +8,6 @@ mod sample_data;
 
 pub use sample_data::sample_data;
 
-#[cfg(feature = "ui")]
-use trustify_module_ui::UI;
-
 use actix_web::{
     body::MessageBody,
     dev::{ConnectionInfo, Url},
@@ -54,7 +51,8 @@ use trustify_module_graphql::RootQuery;
 use trustify_module_importer::server::importer;
 use trustify_module_ingestor::graph::Graph;
 use trustify_module_storage::{service::dispatch::DispatchBackend, service::fs::FileSystemBackend};
-use trustify_module_ui::endpoints::UiResources;
+use trustify_module_ui::{UI, endpoints::UiResources};
+
 use utoipa::OpenApi;
 use utoipa_swagger_ui::SwaggerUi;
 
@@ -98,7 +96,6 @@ pub struct Run {
     #[command(flatten)]
     pub swagger_ui_oidc: SwaggerUiOidcConfig,
 
-    #[cfg(feature = "ui")]
     #[command(flatten)]
     pub ui: UiConfig,
 }
@@ -130,7 +127,6 @@ struct InitData {
     swagger_oidc: Option<Arc<SwaggerUiOidc>>,
     #[cfg(feature = "garage-door")]
     embedded_oidc: Option<embedded_oidc::EmbeddedOidc>,
-    #[cfg(feature = "ui")]
     ui: UI,
     working_dir: Option<PathBuf>,
 }
@@ -217,7 +213,6 @@ impl InitData {
 
         let storage = DispatchBackend::Filesystem(FileSystemBackend::new(storage).await?);
 
-        #[cfg(feature = "ui")]
         let ui = UI {
             // TODO: where/how should we configure these details?
             version: env!("CARGO_PKG_VERSION").to_string(),
@@ -239,7 +234,6 @@ impl InitData {
             storage,
             #[cfg(feature = "garage-door")]
             embedded_oidc,
-            #[cfg(feature = "ui")]
             ui,
             working_dir: run.working_dir,
         })
@@ -325,7 +319,6 @@ fn configure(
         .configure(|svc| {
             // I think the UI must come last due to
             // its use of `resolve_not_found_to`
-            #[cfg(feature = "ui")]
             trustify_module_ui::endpoints::configure(svc, &ui);
         });
 }

--- a/trustd/Cargo.toml
+++ b/trustd/Cargo.toml
@@ -11,7 +11,7 @@ path = "src/main.rs"
 trustify-common = { workspace = true }
 trustify-module-importer = { workspace = true }
 trustify-infrastructure = { workspace = true }
-trustify-server = { workspace = true, features = [ "ui" ] }
+trustify-server = { workspace = true }
 
 anyhow = { workspace = true }
 clap = { workspace = true, features = ["derive", "env"] }

